README.md

```diff
@@ -167,7 +201,7 @@ contains a Terraboard template to automate its installation in Cattle.
 ## Authentication and base URL
 
 Terraboard does not implement authentication. Instead, it is recommended to use
-an authentication proxy such as [oauth2_proxy](https://github.com/bitly/oauth2_proxy).
+an authentication proxy such as [oauth2_proxy](https://github.com/camptocamp/oauth2_proxy).
```

Comment:
To be fair, I think it's the `docker-compose.yml` which should be patched instead. We don't maintain `oauth2_proxy`.

Reply:
OK! Sorry for my mistake.
What do you think about the new proposal? Deleting the oauth proxy?

Reply:
I've updated the docker-compose for 4 reasons:
- To remove the oauth2_proxy.
- To Simplify the bootstrap of the project (reducing the number of necessary variables).
- To make it listen to port 8080 instead of 80.
- To avoid ssl issues with postgres.